docs/src/data/mockServices.tsx

```diff
@@ -0,0 +1,89 @@
+import { Amplify} from 'aws-amplify';
```

Comment:
Formatting error -- i'd ensure your prettier is installed correctly.

1. https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode
2. Make sure you ran `yarn install` at monorepo root

Reply:
+1